vite_ruby/lib/vite_ruby/manifest.rb

```diff
@@ -147,9 +147,11 @@ def resolve_references(manifest)
 
   # Internal: Resolves the manifest entry name for the specified resource.
   def resolve_entry_name(name, type: nil)
-    name = with_file_extension(name.to_s, type)
+    unless name.include?('legacy-polyfills')
```

Comment:
Perhaps we could take this opportunity to generalize the check:

```ruby
unless name.include?("
```

Reply:
I think that a part of your comment didn't come through properly but I think you want to check on the `null` character itself? I think that could work as well. Another solution is to only do this file extension check if a type is explicitly passed. 

But since I wasn't too sure about other effect, I took the minimal change to make it happen. Last time I made a PR, I broke the workflow of other users 🙈 

And maybe also something to take in consideration: If the users using this gem did NOT upgrade vite itself yet. Then this `null` character is not yet in the manifest.

Reply:
Sorry, on my phone, hence the conciseness 😅

What I suggest is changing the condition to check for the presence of the null char in the provided name, skipping the file extension check (since adding the extension is not relevant when looking up virtual entries).

It's true that it wouldn't be compatible with Vite 2.6, but we could bump the recommended Vite version to 2.7, which is [checked on startup](https://github.com/ElMassimo/vite_ruby/blob/0850af2170077bd55d54a725fa6030a32bbbd401/vite_ruby/lib/vite_ruby/config.rb#L86), and would be caught during development or in the CI.